src/Kovan-DssSpell.sol

```diff
@@ -22,271 +22,59 @@ import "dss-exec-lib/DssAction.sol";
-
-        address vat = DssExecLib.vat();
-
-        // Sanity checks
-        require(GemJoinAbstract(collateral.MCD_JOIN).vat() == vat, "join-vat-not-match");
-        require(GemJoinAbstract(collateral.MCD_JOIN).ilk() == collateral.ilk, "join-ilk-not-match");
-        require(GemJoinAbstract(collateral.MCD_JOIN).gem() == collateral.GEM, "join-gem-not-match");
-        require(GemJoinAbstract(collateral.MCD_JOIN).dec() == DSTokenAbstract(collateral.GEM).decimals(), "join-dec-not-match");
-
-        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(
-            collateral.ilk, collateral.PRICE, collateral.DOC, collateral.TAU
-        );
-        (,address pip,,) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(collateral.ilk);
-
-        // Set price feed for RWA-00x
-        DssExecLib.setContract(DssExecLib.spotter(), collateral.ilk, "pip", pip);
-
-        // Init RWA-00x in Vat
-        Initializable(vat).init(collateral.ilk);
-        // Init RWA-00x in Jug
-        Initializable(DssExecLib.jug()).init(collateral.ilk);
-
-        // Allow RWA-00x Join to modify Vat registry
-        DssExecLib.authorize(vat, collateral.MCD_JOIN);
-
-        // Allow RwaLiquidationOracle to modify Vat registry
-        // DssExecLib.authorize(vat, MIP21_LIQUIDATION_ORACLE);
-
-        // set ilk/global DC
-        DssExecLib.increaseIlkDebtCeiling(collateral.ilk, collateral.CEIL, true);
-
-        // No dust
-        // DssExecLib.setIlkMinVaultAmount(collateral.ilk, 0);
-
-        // stability fee
-        DssExecLib.setIlkStabilityFee(collateral.ilk, collateral.RATE, false);
-
-        // collateralization ratio
-        DssExecLib.setIlkLiquidationRatio(collateral.ilk, collateral.MAT);
+    function actions() public override {
 
-        // poke the spotter to pull in a price
-        DssExecLib.updateCollateralPrice(collateral.ilk);
+        // Update early RWA tokens names in ilk registry
+        IlkRegistryAbstract ILK_REGISTRY = IlkRegistryAbstract(DssExecLib.reg());
 
-        // give the urn permissions on the join adapter
-        // DssExecLib.authorize(collateral.MCD_JOIN, collateral.URN);
+        address join;
+        address gem;
+        uint8   dec;
+        uint96  class;
+        address pip;
+        address xlip;
+        string memory name;
+        string memory symbol;
 
-        // set up the urn
-        Hopeable(collateral.URN).hope(collateral.OPERATOR);
+        (, join, gem, dec, class, pip, xlip, name, symbol) = ILK_REGISTRY.ilkData("RWA001-A");
+        ILK_REGISTRY.put("RWA001-A", join, gem, dec, class, pip, xlip, "RWA001-A: 6S Capital", symbol);
 
-        // set up output conduit
-        // Hopeable(collateral.OUTPUT_CONDUIT).hope(collateral.OPERATOR));
+        (, join, gem, dec, class, pip, xlip, name, symbol) = ILK_REGISTRY.ilkData("RWA002-A");
+        ILK_REGISTRY.put("RWA002-A", join, gem, dec, class, pip, xlip, "RWA002-A: Centrifuge: New Silver Series 2 DROP", symbol);
```

Comment:
So, typically the ilk registry uses the `token` and `symbol` of the `gem` associated with the `ilk`.  In the RWA case, we accidentally made these values into an ilk formatted string `SYMBOL-LETTER`.  The downside to doing this is anything that auto-generates the `ilk` from the `gem`'s `symbol` will make `SYMBOL-LETTER-LETTER`.  So this fix is primarily to resolve the `symbol` problem.  We don't use `name` anywhere, although that is usually in the format `RWA-NNN` for these `gems`.  I like the more descriptive names, so I kept those.

The other change is that there is a `file()` function on the ilk registry, so we can save some gas if we use that rather than doing the `ilkData()` => `put()` dance.

```suggestion
        ILK_REGISTRY.file("RWA001-A", "symbol", "RWA001");
        ILK_REGISTRY.file("RWA002-A", "symbol", "RWA002");
        ILK_REGISTRY.file("RWA003-A", "symbol", "RWA003");
        ILK_REGISTRY.file("RWA004-A", "symbol", "RWA004");
        ILK_REGISTRY.file("RWA005-A", "symbol", "RWA005");
        ILK_REGISTRY.file("RWA006-A", "symbol", "RWA006");
```